Document how to drop alert rules (#63)
appuio/openshift4-docs: docs/modules/ROOT/pages/how-tos/monitoring/remove_rules.adoc

= Remove alert rules

[abstract]

> Somtimes an alert rule doesn't makes sense.

Reviewer: ```suggestion
Sometimes an alert rule doesn't make any sense.
```

> Be it that the rule is build on assumptions that don't apply.

Reviewer: ```suggestion
Be it that the rule is built on assumptions that don't apply.
```

Be it that the rule isn't actionable.
If that alert is triggered regularly, it generates a lot of noise and drives us towards alert fatigue.
This how to explains how to get rid of such an alert rule.

1. Identify the name of the alert

   Within Prometheus::
     On the Alerts list, the name is shown within the colored box.
     When an alert is expanded, the name is found within the property `alert`.

   Whithin Alertmanager::
     The alertname is found within the property `alertname`.

2. Add the name to the list of ignored alerts
+
[source,yaml]
----
parameters:
  openshift4_monitoring:
    alerts:
      ignoreNames:
        - MyAlertName
----
+

> Do so on what ever hierarchy level is the most apropriate.

Reviewer: ```suggestion
Do so on whatever hierarchy level is the most appropriate.
```
